<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientCardPage</class>
 <widget class="QWidget" name="clientCardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрационная карта пациента</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="14" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCheckPolicyAffiliation">
     <property name="text">
      <string>Контроль проверки страховой принадлежности</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cmbCheckPolicyAffiliation">
     <item>
      <property name="text">
       <string>Не выполнять</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Мягко</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Строго</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QGroupBox" name="grpPassport">
     <property name="title">
      <string>Визуализация: Паспортные &amp;данные</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="5" column="2">
       <widget class="QCheckBox" name="chkVoluntaryPolisName">
        <property name="text">
         <string>Название полиса ДМС</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="chkVoluntaryPolisNote">
        <property name="text">
         <string>Примечание полиса ДМС</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chkDocOrigin">
        <property name="text">
         <string>Кем выдан документа, удостоверяющего личность</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkDeathDate">
        <property name="text">
         <string>Дата смерти</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chkSNILS">
        <property name="text">
         <string>СНИЛС</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="chkVoluntaryPolicy">
        <property name="text">
         <string>Данные полиса ДМС</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkCompulsoryPolisName">
        <property name="text">
         <string>Название полиса ОМС</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="chkCompulsoryPolisNote">
        <property name="text">
         <string>Примечание полиса ОМС</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="chkBirthTime">
        <property name="text">
         <string>Время рождения</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QGroupBox" name="grpTabWidgets">
     <property name="title">
      <string>Визуализация: Вкладки</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="6" column="0">
       <widget class="QCheckBox" name="chkTabWork">
        <property name="text">
         <string>Вкладка Занятость</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="chkTabAttach">
        <property name="text">
         <string>Вкладка Прикрепление</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkTabSocStatus">
        <property name="text">
         <string>Вкладка Соц.статус</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="chkTabFeature">
        <property name="text">
         <string>Вкладка Особенности</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="chkTabChangeJournal">
        <property name="text">
         <string>Вкладка Журнал изменений</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="chkTabResearch">
        <property name="text">
         <string>Вкладка Обследования</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="chkTabDangerous">
        <property name="text">
         <string>Вкладка Общ. опасность</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="chkTabContingentKind">
        <property name="text">
         <string>Вкладка Контингент</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="chkTabConsent">
        <property name="text">
         <string>Вкладка Согласия</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="chkTabDeposit">
        <property name="text">
         <string>Вкладка Депозитная карта</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="chkTabQuoting">
        <property name="text">
         <string>Вкладка Квоты</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkTabContacts">
        <property name="text">
         <string>Вкладка Прочее</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="chkTabRelations">
        <property name="text">
         <string>Вкладка Связи</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chkTabIdentification">
        <property name="text">
         <string>Вкладка Идентификаторы</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="chkTabMonitoring">
        <property name="text">
         <string>Вкладка Мониторинг</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QCheckBox" name="chkTabEpidCase">
        <property name="text">
         <string>Вкладка ЭпидНаблюдение</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbCheckPolicyAffiliation</tabstop>
  <tabstop>chkDeathDate</tabstop>
  <tabstop>chkBirthTime</tabstop>
  <tabstop>chkSNILS</tabstop>
  <tabstop>chkCompulsoryPolisName</tabstop>
  <tabstop>chkCompulsoryPolisNote</tabstop>
  <tabstop>chkDocOrigin</tabstop>
  <tabstop>chkVoluntaryPolicy</tabstop>
  <tabstop>chkVoluntaryPolisName</tabstop>
  <tabstop>chkVoluntaryPolisNote</tabstop>
  <tabstop>chkTabSocStatus</tabstop>
  <tabstop>chkTabAttach</tabstop>
  <tabstop>chkTabWork</tabstop>
  <tabstop>chkTabFeature</tabstop>
  <tabstop>chkTabChangeJournal</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkVoluntaryPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkVoluntaryPolisName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkVoluntaryPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkVoluntaryPolisNote</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkVoluntaryPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkVoluntaryPolisName</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkVoluntaryPolicy</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkVoluntaryPolisNote</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
